adaptive_navigation/lib/scaffold.dart

```diff
@@ -34,15 +26,17 @@ class AdaptiveNavigationScaffold extends StatefulWidget {
   final List<AdaptiveScaffoldDestination> destinations;
   final ValueChanged<int> onNavigationIndexChange;
   final FloatingActionButton floatingActionButton;
+  final NavigationTypeResolver navigationTypeResolver;
 
-  AdaptiveNavigationScaffold({
-    this.title,
-    this.body,
-    @required this.currentIndex,
-    @required this.destinations,
-    this.onNavigationIndexChange,
-    this.floatingActionButton,
-  })  : assert(currentIndex != null),
+  AdaptiveNavigationScaffold(
+      {this.title,
```

Comment:
Formatting seems a bit off here, I think the previous format was better (braces placement)